modules/http/src/test/java/org/mule/module/http/functional/proxy/HttpProxyTemplateTestCase.java

```diff
@@ -49,11 +59,25 @@
 
     private RequestHandlerExtender handlerExtender;
     private boolean consumeAllRequest = true;
+    private String configFile;
+    private String requestThreadContains;
+
+    @Parameterized.Parameters
+    public static Collection<Object[]> parameters()
+    {
+        return Arrays.asList(new Object[][] {{"http-proxy-template-config.xml","worker"}, {"http-proxy-template-selectors-config.xml", "SelectorRunner"}});
+    }
+
+    public HttpProxyTemplateTestCase(String configFile, String requestThreadContains)
```

Comment:
The variable name is not clear to me, you need to read to code to understand what its used for.